heartbeat/Makefile.am

```diff
@@ -153,6 +153,7 @@ ocf_SCRIPTS	      = AoEtarget		\
 			oraasm			\
 			oracle			\
 			oralsnr			\
+			osceip			\
```

Comment:
You also need to add it to the Makefile.am in doc/man.

Reply:
Done